numba/cuda/tests/cudapy/test_cffi.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import os
+import cffi
+
+from numba import cuda, types
+from numba.cuda.testing import skip_on_cudasim, unittest, CUDATestCase
+
+parent_dir = os.path.dirname(os.path.dirname(__file__))
+data_dir = os.path.join(parent_dir, 'data')
+
+
+@skip_on_cudasim('Simulator does not support linking')
+class TestCFFI(CUDATestCase):
```

Comment:
Does this also need to be skipped on CC=(5,2) devices?

Reply:
I don't think so - the difference between 5.2 and 5.3 is support for `float16`. Could I be missing something here?

Reply:
From OOB conversation. The issue is that the baseline CC is 5,3 and whilst the PTX can be re-targetted for higher versions it cannot be for lower. Suggested fixes are either skip or use 5,0 as the baseline, with the latter being chosen.